tlslite/extensions.py

```diff
@@ -201,6 +213,11 @@ def parse(self, p):
         extType = p.get(2)
         extLength = p.get(2)
 
+        # check if we shouldn't use Certificate extensions parser
+        if self.cert and extType in self._certificateExtensions:
```

Comment:
then it will fall through to universal parser (line 246) , and if it doesn't have a universal parser, it will just save the contents of extension (line 253)